SalesforceFoundation/HEDAP
# Feature/xinnanli/exclude naming

# Critical Changes

# Changes
We've made a change to automatic Household naming to produce the expected behavior. Now, when you add or update a Contact for an existing Household, the original Household name is retained. Note that this functionality requires you to enable Automatic Household Naming in HEDA Settings and to select the "Exclude from Household Name" check box on the Contact.

# Issues Closed
Fixes #591 
# New Metadata

# Deleted Metadata

# Testing Notes
1. Enable automatic household naming in HEDA settings
2. Create several multiple contacts under a household account
3. Insert, update, delete contacts under the household account
4. Make sure the account name displays properly
